scripts/gen-dockerfile.sh

```diff
@@ -8,26 +8,23 @@
 # it is our lowest common denominator in terms of distro support.
 
 # Some commands in this script are structured in order to reduce the number of layers Docker
-# generates. Unfortunately Docker is limited to only 125 layers: 
+# generates. Unfortunately Docker is limited to only 125 layers:
 # https://github.com/moby/moby/blob/a9507c6f76627fdc092edc542d5a7ef4a6df5eec/layer/layer.go#L50-L53
 
 # We require epel packages, so enable the fedora EPEL repo then install dependencies.
 # Install the system dependencies
 # Attempt to clean and rebuild the cache to avoid 404s
 cat <<EOT
 FROM centos:7.6.1810 as builder
-RUN yum clean all && \
-    yum makecache && \
-    yum update -y && \
-    yum install -y epel-release && \
+RUN yum install -y epel-release && \
```

Comment:
Why remove `yum update -y`?

Reply:
`yum install package` will install or update to the latest package, without a package name, it will update all installed packages this is time-consuming.